CONTRIBUTING.md

```diff
@@ -81,4 +83,10 @@ export default ExampleCodeBlock = () => (
 
 To allow customization of pre-built theme components, Docusaurus has a feature called [swizzling](https://docusaurus.io/docs/swizzling), which either allows creating a wrapper around the existing component, or creating a copy of it that can be modified.
 
-It's totally fine to swizzle components, even "unsafe" ones, but take care to minimize changes so that it's easy to upgrade components to their latest versions later on.
\ No newline at end of file
+It's totally fine to swizzle components, even "unsafe" ones, but take care to minimize changes so that it's easy to upgrade components to their latest versions later on.
+
+## License
+
+This project is licensed under the Creative Commons Attribution 4.0 International License (CC BY 4.0) - see the [LICENSE](./LICENSE.md) file for details.
```

Comment:
I don't think we want to license the entire project under CC, that would include things like the Ignite logo. Can we specify recipe contributions are licensed that way? (cc @jamonholmgren)

Reply:
Oh I missed that this licenses the whole project. No, we definitely want to change it to MIT for the project, and CC for contributions.

Reply:
Okay, I was confused. We're able to license contributions under one license and the project under another?

Reply:
Okay, I've updated the License file and stated that contributions are under the CC BY 4.0 license. I'll also update the PR template once that discussions resolved.